eth/peerset.go

```diff
@@ -402,10 +404,27 @@ func (ps *peerSet) waitBscExtension(peer *eth.Peer) (*bsc.Peer, error) {
 		return peer, nil
```

Comment:
output a warn when peer is nil

Reply:
ok, it will be printed as Warning log level in `startProtocols ()`